docs/src/Flex.doc.js

```diff
@@ -1,6 +1,6 @@
 // @flow strict
```

Comment:
@dangerismycat could we add more info on how to use `Flex.Item`?

Reply:
Definitely! I haven't touched the docs or tests aside from fixing them to not be broken — was planning on updating once we've nailed down the implementation. I'll add that to the TODOs above.